tests/components/arve/snapshots/test_sensor.ambr

```diff
@@ -0,0 +1,372 @@
+    'id': <ANY>,
+    'labels': set({
+    }),
+    'name': None,
+    'options': dict({
+    }),
+    'original_device_class': <SensorDeviceClass.TEMPERATURE: 'temperature'>,
+    'original_icon': None,
+    'original_name': 'Temperature',
+    'platform': 'arve',
+    'previous_unique_id': None,
+    'supported_features': 0,
+    'translation_key': 'temperature',
+    'unique_id': 'test-serial-number_temperature',
+    'unit_of_measurement': <UnitOfTemperature.CELSIUS: '°C'>,
+  })
+# ---
+# name: test_sensors[entry_test-serial-number_tvoc]
+  EntityRegistryEntrySnapshot({
+    'aliases': set({
+    }),
+    'area_id': None,
+    'capabilities': None,
+    'config_entry_id': <ANY>,
+    'device_class': None,
+    'device_id': <ANY>,
+    'disabled_by': None,
+    'domain': 'sensor',
+    'entity_category': None,
+    'entity_id': 'sensor.my_arve_tvoc',
+    'has_entity_name': True,
+    'hidden_by': None,
+    'icon': None,
+    'id': <ANY>,
+    'labels': set({
+    }),
+    'name': None,
+    'options': dict({
+    }),
+    'original_device_class': None,
+    'original_icon': None,
+    'original_name': 'TVOC',
+    'platform': 'arve',
+    'previous_unique_id': None,
+    'supported_features': 0,
+    'translation_key': 'tvoc',
+    'unique_id': 'test-serial-number_tvoc',
+    'unit_of_measurement': None,
+  })
+# ---
+# name: test_sensors[my_arve_air_quality_index]
+  StateSnapshot({
+    'attributes': ReadOnlyDict({
+      'device_class': 'aqi',
+      'friendly_name': 'My Arve AQI',
+    }),
+    'context': <ANY>,
+    'entity_id': 'sensor.my_arve_air_quality_index',
+    'last_changed': <ANY>,
+    'last_updated': <ANY>,
+    'state': "<AsyncMock name='Arve().device_sensor_data().aqi' id='4673889600'>",
+  })
+# ---
+# name: test_sensors[my_arve_carbon_dioxide]
+  StateSnapshot({
+    'attributes': ReadOnlyDict({
+      'device_class': 'carbon_dioxide',
+      'friendly_name': 'My Arve CO2',
+      'unit_of_measurement': 'ppm',
+    }),
+    'context': <ANY>,
+    'entity_id': 'sensor.my_arve_carbon_dioxide',
+    'last_changed': <ANY>,
+    'last_updated': <ANY>,
+    'state': "<AsyncMock name='Arve().device_sensor_data().co2' id='4683517632'>",
+  })
+# ---
+# name: test_sensors[my_arve_humidity]
+  StateSnapshot({
+    'attributes': ReadOnlyDict({
+      'device_class': 'humidity',
+      'friendly_name': 'My Arve Humidity',
+      'unit_of_measurement': '%',
+    }),
+    'context': <ANY>,
+    'entity_id': 'sensor.my_arve_humidity',
+    'last_changed': <ANY>,
+    'last_updated': <ANY>,
+    'state': "<AsyncMock name='Arve().device_sensor_data().humidity' id='4674090384'>",
+  })
+# ---
+# name: test_sensors[my_arve_none]
+  StateSnapshot({
+    'attributes': ReadOnlyDict({
+      'friendly_name': 'My Arve TVOC',
+    }),
+    'context': <ANY>,
+    'entity_id': 'sensor.my_arve_none',
+    'last_changed': <ANY>,
+    'last_updated': <ANY>,
+    'state': "<AsyncMock name='Arve().device_sensor_data().tvoc' id='4430967152'>",
```

Comment:
The mock of the arve device should also mock the values of the sensor its using to properly test the sensor